### DISCUSSION TITLE
Extrem high system/Klipper load

### DISCUSSION THREAD

**Discussion by Corleone6993** [Guilouz/Creality-Helper-Script-Wiki#611] [Issues]
Randomly my system load reach 10-11/2 and the Klipper load 400-500%. I dont know why. I make a factory reset. But the Problem is the same

**Comment by grubfriend** [comment]
I've got similar problems. I've tried uninstalling services to reduce load (namely disabling timelapse on everything) but there hasn't been much luck. Had MCU errors/crashes and my prints frequently have 'zits' from the printer pausing from the pad not being able to keep up. I'm going to try printing with the nebula camera unplugged (I think the high video resolution and fps is putting strain on the system, maybe?) and see if that helps at all.
